lib/SIL/IR/SILType.cpp

```diff
@@ -444,6 +444,10 @@ bool SILType::isLoadableOrOpaque(const SILFunction &F) const {
   return isLoadable(F) || !SILModuleConventions(M).useLoweredAddresses();
 }
 
+bool SILType::isLoadable(const SILFunction &F) const {
+  return !isAddressOnly(F) || F.getTypeExpansionContext().isPostAbstraction();
```

Comment:
This is not the right check. The type might still be address-only for other reasons, you don't want to say that _all_ types are loadable here.

Also, `T.isAddressOnly(F)` should be true if and only if `!T.isLoadable(F)`. This violates that invariant.

Reply:
You probably want to make your change somewhere near here: https://github.com/apple/swift/blob/ca91d04968af4cebccfa0259ee1322dcaceea8f2/lib/SIL/IR/TypeLowering.cpp#L2337-L2341